src/main/java/com/google/sps/data/Comment.java

```diff
@@ -1,8 +1,10 @@
 package com.google.sps.data;
 
 import java.util.Objects;
+import com.google.sps.data.Constants;
 
 public final class Comment{
+    private static final double COMMENT_TOXICITY_THRESHOLD = 0.8;  
```

Comment:
this PR is for reactions right? why is there toxicity stuff in here at all?

Reply:
I just have it in here from when I was working on perspective. I'll go ahead and delete